src/main/java/com/wavefront/sdk/direct/ingestion/WavefrontDirectIngestionClient.java

```diff
@@ -405,4 +415,54 @@ public synchronized void close() {
       logger.log(Level.WARNING, "shutdown error", ex);
     }
   }
+
+  /**
+   * Dequeue and return a batch of at most N items from buffer (where N = batchSize), broken into
+   * chunks where each chunk has at most M bytes of data (where M = messageSizeBytes).
+   *
+   * Visible for testing.
+   *
+   * @param buffer            The buffer queue to retrieve items from.
+   * @param batchSize         The maximum number of items to retrieve from the buffer.
+   * @param messageSizeBytes  The maximum number of bytes in each chunk.
+   * @param dropped           A counter counting the number of items that are dropped.
+   * @return A batch of items retrieved from buffer.
+   */
+  static List<List<String>> getBatch(LinkedBlockingQueue<String> buffer, int batchSize,
+                                       int messageSizeBytes, WavefrontSdkCounter dropped) {
```

Comment:
formatting: check the alignment for parameters

Reply:
fixed